src/lamp_py/performance_manager/alerts.py

```diff
@@ -248,6 +248,10 @@ def extract_alerts(
     alerts["severity"] = alerts["severity"].astype("int8")
     alerts["alert_lifecycle"] = alerts["alert_lifecycle"].astype("string")
     alerts["duration_certainty"] = alerts["duration_certainty"].astype("string")
+    alerts["created_timestamp"] = alerts["created_timestamp"].astype("Int64")
+    alerts["last_modified_timestamp"] = alerts["last_modified_timestamp"].astype("Int64")
+    alerts["last_push_notification_timestamp"] = alerts["last_push_notification_timestamp"].astype("Int64")
+    alerts["closed_timestamp"] = alerts["closed_timestamp"].astype("Int64")
```

Comment:
These changes stop the floating point error with `pandas.to_datetime` that we were running into.